helm-charts/strimzi-kafka-operator-helm3/crds/040-Crd-kafka.yaml

```diff
@@ -0,0 +1,3686 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
How will these be kept updated along the existing Helm 2 charts? Are you planning to make any updates to the Makefile to generate/update these in future? (Or write a new Makefile for building the Helm 3 charts?)